src/main/java/name/webdizz/sonar/grammar/utils/SpellCheckerUtil.java

```diff
@@ -0,0 +1,232 @@
+package name.webdizz.sonar.grammar.utils;
+
+import com.swabunga.spell.engine.Configuration;
+import com.swabunga.spell.engine.SpellDictionaryHashMap;
+import com.swabunga.spell.engine.Word;
+import com.swabunga.spell.event.SpellChecker;
+import com.swabunga.spell.event.StringWordTokenizer;
+import name.webdizz.sonar.grammar.exceptions.UnableToLoadDictionary;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Provides utility methods for spell checking. Uses an open source API
+ * <a href="http://jazzy.sourceforge.net/">Jazzy</a> for spell checking.
+ */
+public final class SpellCheckerUtil
```

Comment:
static and final means hard to test code that means code has bad design and smells, let's try to avoid statics and rely on DI